pkg/artifact/ingest_logs.go

```diff
@@ -197,3 +220,37 @@ func (f *logIngester) handleMessage(ctx context.Context, repoName, ghLogsURL, gc
 	}
 	return nil
 }
+
+func (f *logIngester) commentArtifactOnPRs(ctx context.Context, event *EventRecord, artifact *ArtifactRecord) error {
+	logger := logging.FromContext(ctx)
+
+	if artifact.Status != "SUCCESS" {
+		logger.InfoContext(
+			ctx,
+			"skipping PR comment for non-successful log ingestion artifact",
+			"delivery_id", event.DeliveryID,
+		)
+		return nil
+	}
+
+	for _, prNumberStr := range event.PullRequestNumbers {
+		pantheonLogsURL := fmt.Sprintf("https://pantheon.corp.google.com/storage/browser/%s/%s/%s?project=%s", f.bucketName, event.RepositorySlug, event.DeliveryID, f.projectID)
+		comment := fmt.Sprintf("Logs for workflow run [%s](%s) attempt %s uploaded to GCS [here](%s)", event.WorkflowRunID, event.WorkflowURL, event.WorkflowRunAttempt, pantheonLogsURL)
+		prNumber, err := strconv.Atoi(prNumberStr)
+		if err != nil {
+			return fmt.Errorf("error parsing pr number from event payload")
```

Comment:
```suggestion
			return fmt.Errorf("error parsing pr number from event payload: %w", err)
```

You are losing the actual error here.

Reply:
done